src/main/java/seedu/duke/data/notebook/Note.java

```diff
@@ -2,6 +2,12 @@
 
 import java.util.ArrayList;
 
+import static seedu.duke.util.PrefixSyntax.PREFIX_DELIMITER;
+import static seedu.duke.util.PrefixSyntax.PREFIX_TITLE;
+import static seedu.duke.util.PrefixSyntax.PREFIX_TAG;
+import static seedu.duke.util.PrefixSyntax.PREFIX_PIN;
+import static seedu.duke.util.PrefixSyntax.STRING_NEW_LINE;
```

Comment:
Can use Formatter.LS if this is a line separator.

Reply:
Alright thanks will update